src/Style/Display/SwipeableStyle.ts

```diff
@@ -1,15 +1,31 @@
 import styled from 'styled-components';
 
+import { ScreenSize } from '../../Utils/StyleConfig';
 import { SecondaryColor } from '../Colors';
 
 export const SwipeableContainer = styled.div`
   display: flex;
   position: relative;
   overflow-x: auto;
-  scrollbar-width: none; /* hide scrollbar on firefox */
 
+  /* make the horizontal scrollbar visible on desktop */
   &::-webkit-scrollbar {
-    display: none;
+    -webkit-appearance: none;
+    height: 7px;
+  }
+
+  /* make the horizontal scrollbar visible on desktop */
+  &::-webkit-scrollbar-thumb {
+    border-radius: 4px;
+    background-color: rgba(0, 0, 0, 0.5);
```

Comment:
@rosslo btw the color is different from `dst` but that might not matter 
https://gitlab.glints.com/glints/glints-dst/-/blob/staging/next/pages/_app.sc.ts#L78

Reply:
Nice catch, I wasn't aware of this. Just updated it, thanks!